externs/lcevc.js

```diff
@@ -0,0 +1,93 @@
+/**
+* @fileoverview Externs for LcevcDil
+* compiler.
+*
+* @externs
+*/
+'use strict';
+
+var libDPIModule = {};
+var LcevcDil = {};
+
+/**
+* LCEVC DIL constructor
+* @constructor
+*/
+LcevcDil.LcevcDIL = class {
+  /**
+     *  @param {HTMLVideoElement} media
+     *  @param {HTMLCanvasElement} canvas
+     *  @param {Object=} dilConfig
```

Comment:
I think we should be more specific about this.  `Object` should be avoided if we can instead define the fields that are expected and what types they should contain.

Optional fields can be unions with `undefined`.  For example: `{(number|undefined)}`

Reply:
Created a more specific type for the dilConfig in externs/lcevc.js - LcevcDil.LcevcDilConfig which is used for any further references of the dilConfig